tools/minibrowser/build.gradle

```diff
@@ -1,8 +1,20 @@
+buildscript {
+    repositories {
+        google()
```

Comment:
This should be defined in the root settings.gradle file (and google repo is already there so there should not be anything to add in fact).

Reply:
oh ok, not sure why I added this